runtime/server_http_request.go

```diff
@@ -30,31 +30,25 @@ import (
 	"time"
 
 	"github.com/julienschmidt/httprouter"
-	"github.com/uber-go/tally"
 	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 )
 
 // ServerHTTPRequest struct manages request
 type ServerHTTPRequest struct {
 	httpRequest *http.Request
 	res         *ServerHTTPResponse
-	gateway     *Gateway
 	started     bool
 	startTime   time.Time
-	metrics     *EndpointMetrics
+	metrics     *InboundHTTPMetrics
 	queryValues url.Values
 	parseFailed bool
-
-	Logger *zap.Logger
-	Scope  tally.Scope
-
-	EndpointName string
```

Comment:
it's useful to keep the `EndpointName` and `HandlerName` for future debugging.

Reply:
Done.